docker/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+docker-compose --env-file ./.env -f docker-compose.prod.yml run api python manage.py migrate
+docker-compose --env-file ./.env -f docker-compose.prod.yml up -d
+docker system prune -af
```

Comment:
why do we need this part

Reply:
I used it to remove old containers since everytime the app is deployed a new container for the first command is spun up but never removed. The first command migrates any schema changes to the database but there's probably a better way of running it without the need to force prune